### PR TITLE
cmd/atlas: add verbose steps information to lint command

### PR DESCRIPTION
**Pull request by a8m**

### CODE REVIEW

File: cmd/atlas/internal/ci/ci_test.go [hunk 1]
```diff
@@ -177,6 +182,10 @@ type testDir struct {
 	files []migrate.File
 }
 
+func (t testDir) Path() string {
+	return "migrations"
+}
+
```
Comment (masseelch) [inline cmd/atlas/internal/ci/ci_test.go:188]: :rocket:

**Review by masseelch — APPROVED** [review]

File: cmd/atlas/internal/ci/run.go [hunk 2]
```diff
@@ -35,50 +36,80 @@ type Runner struct {
...
+	stepLoadChanges    = "Replay Migration Files"
+	stepAnalyzeFile    = "Analyze %s"
+)
+
+func (r *Runner) summary(ctx context.Context) error {
+	r.sum = NewSummaryReport(r.Dev, r.Dir)
+
+	// Integrity check.
+	switch err := migrate.Validate(r.Dir); {
+	case errors.Is(err, migrate.ErrChecksumNotFound):
```
Comment (ronenlu) [inline cmd/atlas/internal/ci/run.go:68]: what happens if err is migrate.ErrChecksumNotFound?

**Review by ronenlu — COMMENTED** [review]
pls address small comment

File: cmd/atlas/internal/ci/run.go (hunk 2 above)
Comment (a8m) [reply cmd/atlas/internal/ci/run.go:68]: This step is skipped

**Pull request merged**